.github/workflows/build_and_test.yaml

```diff
@@ -166,3 +150,14 @@ jobs:
             fi
             sleep 30
           done
+          ./gradlew -DjacocoBuild=1 jacocoTestReport
```

Comment:
is some of the originally existed option needed here? e.g. `-PskipTestGroup=disabledOnCI`